src/index.html

```diff
@@ -1,13 +1,42 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div 
+    class="card" 
+    data-qa="card"
+    >
+      <img 
+      src="./images/imac.jpeg" 
+      class="card__img" 
+      alt="IMac"
+      >
```

Comment:
```suggestion
      <img 
        src="./images/imac.jpeg" 
        class="card__img" 
        alt="IMac"
      >
```
follow this code style (all over the code) 😉